packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -1271,17 +1273,37 @@ class RenderBackdropFilter extends RenderProxyBox {
   @override
   bool get alwaysNeedsCompositing => child != null;
 
+  // Computes the bounds of this render object in the coordinate space where the
+  // filter is applied.
+  //
+  // This method applies all ancestor transforms from this render object up to
+  // the root of the render tree, including the root transform of [RenderView],
+  // and then excludes the offset used for the pushed layer. The resulting
+  // transform is applied to the rectangle that represents this object's size.
+  Rect _sizeForFilter(Offset offset) {
+    final Matrix4 transform = Matrix4.translationValues(-offset.dx, -offset.dy, 0);
+    for (RenderObject current = this; current.parent != null; current = current.parent!) {
```

Comment:
you can use getTransformTo to get the global transform

Reply:
If I remember correctly I've tried it, but it doesn't include the root transform of `RenderView`. Rather than combining `getTransformTo` with the root transform, I think it's easier to compute everything this way.